azkaban-common/src/main/java/azkaban/executor/JdbcExecutorLoader.java

```diff
@@ -773,6 +776,205 @@ private Connection getConnection() throws ExecutorManagerException {
     return connection;
   }
 
+
+  /**
+   *
+   * {@inheritDoc}
+   *
+   * @see azkaban.executor.ExecutorLoader#fetchActiveExecutors()
+   */
+  @Override
+  public List<Executor> fetchActiveExecutors() throws ExecutorManagerException {
```

Comment:
Does this method return null or empty list when there are not matches?  Please add documentation about this behavior.